### PR TITLE
Additional checks amount output

### PR DESCRIPTION
**Pull request by BibianaC**
relates to #1200 

- Quality section should always be numbers.
- Usefulness section % unless the result is below 5%, in which case it switches to numbers.

### CODE REVIEW

File: cove_360/lib/threesixtygiving.py [hunk 1]
```diff
@@ -285,13 +285,14 @@ def common_checks_360(context, upload_dir, json_data, schema_obj):
...
+            json_data, cell_source_map, TEST_CLASSES[test_class_type], ignore_errors=True, return_on_error=None)
+
         context.update({
-            '{}_errored'.format(test_classes_type): extra_checks is None,
-            '{}_checks'.format(test_classes_type): extra_checks,
-            '{}_checks_count'.format(test_classes_type): (len(extra_checks) if extra_checks else 0) + (1 if context['data_only'] else 0)
+            '{}_errored'.format(test_class_type): extra_checks is None,
+            '{}_checks'.format(test_class_type): extra_checks,
+            '{}_checks_count'.format(test_class_type): (len(extra_checks) if extra_checks else 0) + (1 if context['data_only'] else 0)
         })
```
Comment (michaelwood) [inline cove_360/lib/threesixtygiving.py:297]: Is this related to "quality checks in numbers" ? if not it is a good refactor that could be in a separate commit.

File: cove_360/lib/threesixtygiving.py [hunk 2]
```diff
@@ -911,7 +918,7 @@ def process(self, grant, path_prefix):
                         self.json_locations.append(id_location)
                         self.count += 1
 
-        self.heading = self.format_heading_count(self.check_text['heading'])
+        self.heading = self.format_heading_count(self.check_text['heading'], test_class_type='quality_accuracy')
         self.message = self.check_text['message'][self.grants_percentage]
```
Comment (michaelwood) [inline cove_360/lib/threesixtygiving.py:926]: A "nice to have" would if we could start using a const for this value instead of typing it out each time.

File: cove_360/lib/threesixtygiving.py [hunk 3]
```diff
@@ -997,6 +1004,7 @@ def process(self, grant, path_prefix):
 def run_extra_checks(json_data, cell_source_map, test_classes):
     if 'grants' not in json_data:
         return []
+
     test_instances = [test_cls(grants=json_data['grants']) for test_cls in test_classes]
```
Comment (michaelwood) [inline cove_360/lib/threesixtygiving.py:1011]: extra line?

File: cove_360/lib/threesixtygiving.py (hunk 2 above)
Comment (BibianaC) [reply cove_360/lib/threesixtygiving.py:929]: Really good comment, thanks! I didn't like having so much repetition but it didn't come to do it as simple as adding a constant.

File: cove_360/lib/threesixtygiving.py (hunk 3 above)
Comment (BibianaC) [reply cove_360/lib/threesixtygiving.py]: I have deleted it.

File: cove_360/lib/threesixtygiving.py (hunk 1 above)
Comment (BibianaC) [reply cove_360/lib/threesixtygiving.py:300]: You are right, this should had been in a different commit. I will leave it as it is now if you don't mind.

**Comment by BibianaC** [comment]
[Quality feedback messages in 360Giving Quality Checks should give number of grants, not %, if it's a small number](https://trello.com/c/ttCKtbdv/43-quality-feedback-messages-in-360giving-quality-checks-should-give-number-of-grants-not-if-its-a-small-number)
